lib/streamlit/elements/lib/subtitle_utils.py

```diff
@@ -161,7 +161,7 @@ def process_subtitle_data(
         raise TypeError(f"Invalid binary data format for subtitle: {type(data)}.")
 
     if runtime.exists():
-        filename = hashlib.md5(label.encode()).hexdigest()
+        filename = calc_md5(label.encode())
```

Comment:
this is the only functional change, right? All other diffs are coming from refactoring by setting `usedforsecurity` directly instead of destructuring it from `HASHLIB_KWARGS`?

Reply:
I think there are a few more places where I used `calc_md5` instead of direct usage of `hashlib` since  `calc_md5` is just a convenient method to calculate a hex digest based on string or bytes. But there are a few other usages where its not as easy to directly use the utils method, which I changed to explicitly use `usedforsecurity`.